incident_archive/ir_version_2_0.md

```diff
@@ -0,0 +1,169 @@
+# Incidents
+Incidents can happen. Things do not always go as planned, and that can be okay. However, when incidents occur, the underlying issue (i.e., root cause) should be identified and remediated to discourage the incident from occurring again. Formally documenting the incident in a report encourages an understanding of all contributing root cause(s), and it presents the opportunity to clearly communicate a remediation plan to reduce the probability of its reoccurrence. 
+
+Depending on the root programs in which a CA Owner participates, it may be required to:
+* create [Audit Incident Reports](incident-report#audit-incident-reports) when audits have non-conformities, qualifications, or modified opinions.
```

Comment:
While this probably warrants a different discussion, we wanted to highlight some issues we see with the Audit Incident Reports. Quite often (not always), we see that these combine several incidents within the same report, with a short explanation of the issue and root cause, rather than a full incident report. 

We instead have always taken the approach that any incident discovered during an audit is still an incident in and of its own and thus needs a Preliminary Incident Report within 72 hours. 

If any issue discovered is a new incident, we do wonder if there’s use still for an Audit Incident Report, unless the modified opinion itself is to be considered a different incident, versus the issues the modified opinion is based on. Is there any specific allowance for not creating an incident report if an issue is discovered during an audit, and instead wait until an Audit Incident Report can be filed?